File: chrcode20.js
// Character Code List DEC | IchigoLatte 1.0.0b8+
// Copyright (c) 2019 Keiichi Shiga (BALLOON | FU-SEN)
// The MIT License (MIT) - https://gist.github.com/fu-sen/7e55ee56a67e07d804db9379178e3cb5

//Character Code List 20

var i=0;

cls();

while(i<256)
{
  lc(i%20+4,i/20+3);
  log(chr(i));

  i=i+1;
}

i=0;

while(i<20)
{
  lc(i+4,0);
  if(i/10==0){log(" ");}
         else{log(i/10);}

  lc(i+4,1);
  log(i%10);

  lc(i+4,2);
  log(chr(145));

  i=i+1;
}

i=0;

while(i<256)
{
  lc(0,i/20+3);
  if(i<10){log(" ");}
  if(i<100){log(" ");}
  log(i);
  log(chr(146));

  i=i+20;
}

lc(3,2); log(chr(152));
lc(2,1); log("\\");
lc(1,0); log("\\");

lc(0,20);


File: chrcode20.rb
# IchigoLatte character code list DEC | IchigoLatte Ruby
# Copyright (c) 2019 BALLOON | FU-SEN
# The MIT License (MIT) - https://mit.balloon.net.eu.org/#2019

# Chratcter list 20

cls

c=0
while c<256
 lc c%20+4,c/20+3
 print chr c
 c=c+1
end

c=0
while c<20
 if c>9
  lc c+4,0
  print "1"
 end
 lc c+4,1
 print c%10
 lc c+4,2
 print chr 145
 c=c+1
end

c=0
while c<256
 lc 0,c/20+3
 if c==0
  print chr 32
 end
 if c<100
  print chr 32
 end
 print c
 print chr 146
 c=c+20
end

lc 1,0
print "\\"
lc 2,1
print "\\"
lc 3,2
print chr 152

lc 0,20
